### ISSUE TITLE
add support for category_encoder 

### ISSUE THREAD

**Issue by peyman-spring** [onnx/sklearn-onnx#668]
In my experience, category_encoder (https://contrib.scikit-learn.org/category_encoders/index.html) is one of the most useful sklearn contrib libraries to deal with categorical features. Would it be possible to add support for this library? 
I tried to write a custom convertor but didn't have any luck making it work :/

**Comment by xadupre** [comment]
We won't add a converter for this library in sklearn-onnx as it is not part of scikit-learn. Would that be ok if I add one example in the documentation to show how to create a custom converter? In that case, which encoder would like me to do?

**Comment by peyman-spring** [comment]
that would be amazing!! This is the encoder I am working with `category_encoders.woe.WOEEncoder`
https://contrib.scikit-learn.org/category_encoders/woe.html#

**Comment by peyman-spring** [comment]
Hi @xadupre, I was wondering if you have added the example for `category_encoders.woe.WOEEncoder`? appreciate your help!

**Comment by xadupre** [comment]
Sorry for the delay. I looked into the code WOEEncoder and the conversion is not a simple one. I did not have time to do it. I'll try to have something by end of this month.

**Comment by xadupre** [comment]
Let me know if PR #720 answers your need. I created a transformer WOETransformer(intervals, labels, onehot=True/False) which returns a label associated to an interval a feature falls into. I'll add an example in the documentation later. I would convert the WOEEncoder from category_encoder to this WOETransformer and calls the converter on that model.

**Comment by xadupre** [comment]
The PR was merged. The full example is here: https://github.com/onnx/sklearn-onnx/blob/master/docs/tutorial/plot_catwoe_transformer.py. It currently require the development version but it should be released soon. I'll close the issue. Feel free to reopen it.
